cvmfs/cloud_testing/run.sh

```diff
@@ -242,6 +247,12 @@ devel_package=$(read_package_map    ${ctu}/pkgmap "$platform" 'devel'    )
 unittest_package=$(read_package_map ${otu}/pkgmap "$platform" 'unittests')
 config_packages="$(read_package_map ${otu}/pkgmap "$platform" 'config'   )"
 
+# if we are on mac then we have to run an special script
```

Comment:
It might appear to be more natural, but delaying the forking until this point allows the original script to collect the different variables that later the mac script will need. The fork is produced after getting the packagemap file, processing it and reading the variables, and all this process would anyway have to be copied to the other script. I think it's a good point to fork